tempus_fugit_minecraft/player.py

```diff
@@ -111,8 +113,9 @@ def get_motion_vector(self) -> tuple:
             dx = 0.0
             dz = 0.0
         return dx, dy, dz
-
-    def speed_up(self) -> None:
+ 
+    """Issue #71 """
+    def increase_speed(self) -> None:
```

Comment:
Similar to above since this is getting renamed, `increase_walk_speed`

Reply:
> Similar to above since this is getting renamed, `increase_walk_speed`

Done!